packages/telemetry/README.md

```diff
@@ -0,0 +1,3 @@
+# Astro Telemetry
+
+TODO
```

Comment:
safe to remove? I don't think we really maintain these src/ READMEs well, I'd rather put the docs in code